src/libraries/Microsoft.PowerFx.Core/Public/Types/BlankType.cs

```diff
@@ -15,9 +15,9 @@ internal BlankType()
         {
         }
 
-        public override void Visit(ITypeVisitor vistor)
+        public override void Visit(ITypeVisitor visitor)
```

Comment:
>visitor [](http://example.com/codeflow?start=48&length=7)

wow, impressed it caught this.

Reply:
So do I
[https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1725](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1725)